tests/Relay/Connection/AbstractConnectionBuilderTest.php

```diff
@@ -17,11 +17,11 @@ abstract class AbstractConnectionBuilderTest extends TestCase
     protected function getExpectedConnection(array $wantedEdges, $hasPreviousPage, $hasNextPage): ConnectionInterface
     {
         $edges = [
-            'A' => new Edge('YXJyYXljb25uZWN0aW9uOjA=', 'A'),
```

Comment:
Because the cursor is now encoded by default to the url-safe version of base64 format, so that it can be safely passed around as a query param for example

Reply:
this is not Relay compliant (compared to the js reference implementation), we should maybe have both encoders using base64 ("plain" and url-safe) but use the old behavior as default. What do you think?

Reply:
As far as I read neither the Relay specs nor the GraphQL specs mention anything about the url-safe version not being compliant. The definition of the cursor is:

>IDs are designed to be opaque (the only thing that should be passed to the id argument on node is the unaltered result of querying id on some object in the system), and base64ing a string is a useful convention in GraphQL to remind viewers that the string is an opaque identifier.

To me it looks safer to default to a url-safe version of base64 and I think that almost any JS library are able to decode it, but if you feel safer then I'm totally fine with reverting the default to RFC 4648

Reply:
let's keep it so but this should also be describe in UPGRADE-1.0.md file.

Reply:
In the end I decided to revert the default encoding to plain base64 so that existing applications that cannot decode the url-safe format won't break